scripts/get_summary.py

```diff
@@ -0,0 +1,97 @@
+import os
+import statistics
+import sys
+from datetime import datetime, timedelta
+from glob import glob
+
+
+def get_results(folders, number_of_days=None):
+    results_list = []
+    date_valid_from = (
+        (datetime.utcnow() - timedelta(days=number_of_days)) if number_of_days else None
+    )
+
+    for folder in folders:
+        date = folder.split('/')[-1].split('T')[0]
+        if date_valid_from and datetime.strptime(date, "%Y-%m-%d") < date_valid_from:
+            continue
+
+        get_request_response_times = []
+        post_request_response_times = []
+        all_response_times = []
+
+        for file in glob(folder + '/*stats.csv'):
+
+            with open(file) as f:
+                data = f.read()
+
+            get_values = []
+            post_values = []
+
+            for line in data.split('\n'):
+                if 'Name' in line:
+                    continue
+
+                values = line.split(',')
+
+                percentile_99th = int(values[18])
+                if values[1].startswith('"/questionnaire'):
+                    if values[0] == '"GET"':
+                        get_values.append(percentile_99th)
+                    elif values[0] == '"POST"':
+                        post_values.append(percentile_99th)
+
+            get_request_response_times.extend(get_values)
+            post_request_response_times.extend(post_values)
+
+            all_response_times = get_values + post_values
+
+        results_list.append(
+            [
+                date,
+                statistics.mean(get_request_response_times),
+                statistics.mean(post_request_response_times),
+                statistics.mean(all_response_times),
+            ]
+        )
+
+    return results_list
+
+
+def validated_environment_variables():
```

Comment:
I think it would be clearer if this method was named after what it does, rather than what it returns, possibly `parse_environment_variables`?